src/service/email.service.js

```diff
@@ -0,0 +1,60 @@
+import 'dotenv/config';
+import nodemailer from 'nodemailer';
+
+const transporter = nodemailer.createTransport({
+  host: process.env.SMTP_HOST,
+  port: process.env.SMTP_PORT,
+  secure: false, // true for port 465, false for other ports
```

Comment:
Consider setting `secure` to `true` if you are using port 465 for SMTP, as it is the standard port for secure SMTP connections. Ensure that the correct port and security settings are used based on your SMTP provider's requirements.